go/private/rules/library.bzl

```diff
@@ -191,8 +194,9 @@ go_library = rule(
         "_allowlist_function_transition": attr.label(
             default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
         ),
-    },
-    toolchains = [GO_TOOLCHAIN],
+    } | CGO_ATTRS,
+    fragments = CGO_FRAGMENTS,
```

Comment:
probably want to update https://github.com/bazel-contrib/rules_go/blob/36d0bcbdcbba5875a828e87ff37e5de74d95667e/go/toolchains.rst#writing-new-go-rules to reflect the new recommendations?